docs/source/developers/index.rst

```diff
@@ -15,6 +15,10 @@
 .. specific language governing permissions and limitations
 .. under the License.
 
+****************************
+Development and Contributing
+****************************
```

Comment:
Did you check that this gives a good result? (or, what was the reason to add this?)

The reason I am not sure this is a good change, is because in the main `index.rst`, we only list `developers/index`, which expands to the mulitple titles in the html docs. But by adding this title, it might end up in only showing the above title.

Reply:
Ah, I see in your description this was for the breadcrumbs, those indeed don't look good. 
But, then, I think we might also want to update the last toctree in  `index.rst` to list the different sections (similarly to how in the first toctree the different 'format' sections are also listed explicitly)

Reply:
Yes, this was to try to fix the breadcrumbs, which currently look like `Docs » <no title> » Contribution Guidelines`. I'll look into the index.rst.